### DISCUSSION TITLE
Issue with Facerecognition Community Container - service not accesible (Imaginary)

### DISCUSSION THREAD

**Discussion by elephantastyczny** [nextcloud/all-in-one#4625] [Questions] [answered]
Hi All,

I've a problem with comunity container - facerecognition

Nextcloud AIO 8.2.1 - NC29

All containers running without issues
But facerecognition doesn't work.
When I try to run it manually 

sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ face:background_job -u **user_id**

I got following error

1/8 - Executing task CheckRequirementsTask (Check all requirements)
        An Imaginary service (http://nextcloud-aio-imaginary:9000) was configured to manage temporary images, but it is inaccessible.Check out the service, or set the 'preview_imaginary_url' key appropriately.
Task CheckRequirementsTask signalled we should not continue, bailing out

**Comment by elephantastyczny** [comment]
Logs from Imaginary container
Imaginary has started
172.19.0.9 - - [07/May/2024 09:08:37] "GET / HTTP/1.1" 401 53 0.0016

**Comment by Bikholf** [comment]
i am facing the same issue. when looking at the Imaginary logs, iseems like the nextcloud container has no troubles connection to the Imaginary one (changed parts of ip adresss with X and excluded unnecessary parts):
`192.168.XXX.6 - - [09/May/2024 16:07:18] "POST [...] HTTP/1.1" 200 251507 2.5698`

But when i asm trying to test tzhe face recognition with sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ face:background_job -u user_id the logs say:
`192.168.XXX.4 - - [09/May/2024 16:19:54] "GET / HTTP/1.1" 401 53 0.0004`

**Comment by szaimen** [comment, marked as answer]
I think the reason is that facerecognition does not support the imaginary key yet. It was implemented with https://github.com/matiasdelellis/facerecognition/pull/746 but is not released yet iirc.

**Comment by addy419** [comment]
Same issue here, but I guess we need to wait for the patch simon [mentioned](https://github.com/matiasdelellis/facerecognition/pull/746)

**Comment by matiasdelellis** [comment]
I just uploaded the update enabling NC29 and the patch mentioned. Try it. 😉
